src/test/java/net/sf/jsqlparser/statement/select/SelectTest.java

```diff
@@ -1480,20 +1480,20 @@ public void testCase() throws JSQLParserException {
 
         statement =
                 "SELECT a, (CASE " + "WHEN (CASE a WHEN 1 THEN 10 ELSE 20 END) > 15 THEN 'BBB' " + // "WHEN
-                                                                                                   // (SELECT
-                                                                                                   // c
-                                                                                                   // FROM
-                                                                                                   // tab2
-                                                                                                   // WHERE
-                                                                                                   // d
-                                                                                                   // =
-                                                                                                   // 2)
-                                                                                                   // =
-                                                                                                   // 3
-                                                                                                   // THEN
-                                                                                                   // 'AAA'
-                                                                                                   // "
-                                                                                                   // +
+                // (SELECT
```

Comment:
please just remove that (I know, its not related to your work, but help with housekeeping is appreciated).